frontend/src/app/workflows/model/helper/workflows.schema.ts

```diff
@@ -0,0 +1,198 @@
+export const workflowsSchema =
```

Comment:
Shouldn't this be fetched from schemastore.org?

Reply:
I will have to do a bit more research on this combined with ajv